### PR TITLE
MTP-1473 focus remove search

### PR DESCRIPTION
**Pull request by lha047**
- fix focus on items in header
- fix focus on breadcrumbs
- fix search outside of view on mobile

### CODE REVIEW

File: src/svelte/components/form/Search.svelte [hunk 1]
```diff
@@ -10,8 +11,16 @@
   export let searchButtonText: string | undefined
   export let placeholder: string | undefined
   export let ariaControls: string | undefined
+  export let ariaRemoveTextLabel = 'Tøm'
   export let inputClass = ''
-  export let primaryButton = false
+  let isInitialized = false
+
+  beforeUpdate(() => {
+    if ((value === undefined || value === '') && !isInitialized && document) {
```
Comment (amundlindstol) [inline src/svelte/components/form/Search.svelte:19]: (value === undefined || value === '') -> !value
... but i think you can remove the check entirely? isInitialized should be enough

Comment (amundlindstol) [reply src/svelte/components/form/Search.svelte:19]: also... can document be undefined?

**Review by amundlindstol — APPROVED** [review]
LGMT :D
See comments :)

File: src/svelte/components/form/Search.svelte (hunk 1 above)
Comment (petterkoding) [reply src/svelte/components/form/Search.svelte:19]: What does beforeUpdate do in this situation?

Isn't it possible to not use it at all and just check {#if value} or {#if value.length > 0} on line 49?

**Review by petterkoding — APPROVED** [review]

**Pull request merged**